doozerlib/cli/release_gen_assembly.py

```diff
@@ -369,6 +390,9 @@ def exit_with_error(msg):
             'extras': -1,
             'metadata': -1,
         }
+        if not date:
+            date = "YYYY-Mon-DD"
```

Comment:
Would like to use a real date as the placeholder rather than a format string `YYYY-Mon-DD`, e.g. `1970-Jan-01`. A real date will make validation easier.

Or just make this field required.